axelrod/strategies/hunter.py

```diff
@@ -8,7 +8,12 @@
 
 
 class DefectorHunter(Player):
-    """A player who hunts for defectors."""
+    """A player who hunts for defectors.
+
+    Names:
+
+    Defector Hunter: Original name by Marc Harper
```

Comment:
These hunters are Karol I think except for cycle hunter and eventual cycle hunter